ripd/rip_zebra.c

```diff
@@ -180,9 +180,6 @@ void rip_redistribute_clean(void)
 						VRF_DEFAULT);
 
 		vrf_bitmap_unset(zclient->redist[AFI_IP][i], VRF_DEFAULT);
-
-		/* Remove the routes from RIP table. */
-		rip_redistribute_withdraw(i);
 	}
```

Comment:
Fair question. The commit message explains why this call was removed:

> * rip_redistribute_clean() is only called on shutdown, so there's
  no need to call rip_redistribute_withdraw() there since the RIP
  table is already cleaned up elsewhere.

Reply:
thanks.